README.md

```diff
@@ -1,28 +1,45 @@
 ## <img src="https://fnproject.io/images/fn-300x125.png" alt="Fn Project Logo" height="21" width="50"> Project Documentation
 
-If you are new to Fn, check out this [introduction](fn/general/introduction.md). 
+## Welcome
+Fn is an event-driven, open source, [Functions-as-a-Service (FaaS)](https://github.com/fnproject/docs/blob/master/fn/general/introduction.md) compute platform that you can run anywhere. Some of its key features:
+
+* Open Source
+* Native Docker: use any Docker container as your Function
+* Supports all languages
+* Run anywhere
+  * Public, private and hybrid cloud
+  * Import Lambda functions and run them anywhere
```

Comment:
this has to be tested, I'd rather ensured if it really works and then posts it.

Reply:
yea i'm about 100% sure these are broken (they don't support http-stream format)

Reply:
we should use node-fdk, and fix them up, but anyway, yea, agree... broken, shouldn't promote

Reply:
removed.